src/mongocxx/options/find_one_and_delete.cpp

```diff
@@ -20,6 +20,10 @@ namespace mongocxx {
 MONGOCXX_INLINE_NAMESPACE_BEGIN
 namespace options {
 
+void find_one_and_delete::max_time_ms(std::int64_t max_time_ms) {
```

Comment:
This is another place I think we should consider making the parameter type a std::chrono, and dropping the 'ms' from the method name. As before, not to be changed here, but lets write up the ticket.